recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "aiohttp" %}
-{% set version = "3.4.4" %}
-{% set sha256 = "51afec6ffa50a9da4cdef188971a802beb1ca8e8edb40fa429e5e529db3475fa" %}
+{% set version = "3.5.1" %}
+{% set sha256 = "c115744b2a0bf666fd8cde52a6d3e9319ffeb486009579743f5adfdcf0bf0773" %}
 
```

Comment:
do not merge... upstream added a dependency on typing_extensions which is currently not solvable in py36.